src/main/java/com/mateacademy/table/JpaUtil.java

```diff
@@ -0,0 +1,44 @@
+package com.mateacademy.table;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.cfg.Configuration;
+import org.hibernate.cfg.Environment;
+import org.hibernate.service.ServiceRegistry;
+
+import java.util.Properties;
+
+public class JpaUtil {
```

Comment:
Utility class should be final and have a private constructor.
If you use Lombok look here https://projectlombok.org/features/experimental/UtilityClass